src/core/batch/batch.ts

```diff
@@ -17,17 +21,49 @@ interface BloomFilterData {
   RowGroupIndex: number;
 }
 
+interface WriteBatchFileOptions {
+  bloomFilters?: BloomFilterOptions;
+}
+
 export type BatchFileObject = string;
 
 /**
  * createFile() takes a series of DSNP messages and returns a Batch file
- * object for publishing. This method is not yet implemented.
+ * object for publishing.
  *
- * @param events  An array of DSNPEvents to include in the batch file
- * @returns         A batch file object
+ * @param messages An array of DSNPMessage to include in the batch file
+ * @returns         A URL
```

Comment:
this could be clearer, because it says it returns a Batch file object above when it returns a URL to the file, right?